packages/react-sdk/src/components/image.stories.tsx

```diff
@@ -1,18 +1,145 @@
-import React from "react";
+import React, { type ComponentProps } from "react";
 import type { ComponentMeta, ComponentStory } from "@storybook/react";
 import { Image as ImagePrimitive } from "./image";
 import argTypes from "./image.props.json";
+import { css, CSS } from "../stitches";
 
 export default {
   title: "Components/Image",
   component: ImagePrimitive,
   argTypes,
 } as ComponentMeta<typeof ImagePrimitive>;
 
-const Template: ComponentStory<typeof ImagePrimitive> = (args) => (
-  <ImagePrimitive {...args} />
-);
+type ImageProps = ComponentProps<typeof ImagePrimitive>;
 
-export const Image = Template.bind({});
+const ImageTemplate: ComponentStory<
+  React.ForwardRefExoticComponent<ImageProps & { css: CSS }>
+> = (args) => {
+  const className = css({ maxWidth: "100%" })({
```

Comment:
We discussed that we will be moving away from stitches on canvas long term in favor of something statically extractable,  so it would be good if we didn't have to rewrite stories, its currently only used in one place.

Alternative would be to use style prop, would that work for you?

Reply:
Done